addons/br/dfe/extensions.go

```diff
@@ -0,0 +1,182 @@
+package dfe
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/pkg/here"
+)
+
+// Brazilian extension keys for fiscal documents
+const (
+	ExtKeyFiscalIncentive = "br-dfe-fiscal-incentive"
+	ExtKeyMunicipality    = "br-dfe-municipality"
+	ExtKeySimples         = "br-dfe-simples"
+	ExtKeySpecialRegime   = "br-dfe-special-regime"
+)
+
+var extensions = []*cbc.Definition{
+	{
+		Key: ExtKeyFiscalIncentive,
+		Name: i18n.String{
+			i18n.EN: "Fiscal Incentive",
+			i18n.PT: "Incentivo Fiscal",
+		},
+		Values: []*cbc.Definition{
+			{
+				Code: "1",
+				Name: i18n.String{
+					i18n.EN: "Has incentive",
+					i18n.PT: "Possui incentivo",
+				},
+			},
+			{
+				Code: "2",
+				Name: i18n.String{
+					i18n.EN: "Does not have incentive",
+					i18n.PT: "Não possui incentivo",
+				},
+			},
+		},
+		Desc: i18n.String{
+			i18n.EN: here.Doc(`
+				Indicates whether a party benefits from a fiscal incentive.
+			`),
+		},
+		Sources: []*cbc.Source{
+			{
+				Title: i18n.String{
+					i18n.EN: "NFS-e ABRASF Taxpayer Guidance Manual (v2.04)",
+					i18n.PT: "NFS-e ABRASF Manual de Orientação do Contribuinte (v2.04)",
+				},
+				URL: "https://abrasf.org.br/biblioteca/arquivos-publicos/nfs-e-manual-de-orientacao-do-contribuinte-2-04/download",
```

Comment:
This suggests that the codes are specific for the NFS-e, is there another source for the NF-e?

Reply:
@samlown this is a boolean value indicating whether a party qualifies for having a fiscal incentive  according to Lei nº 11.196/2005 (Lei do Bem) (i.e. the company does R&D). A field for this exists both in NFS-e and NF-e and it has the same values ("1" or "2"). I imagine this could also appear in other document types in the future, potentially with different values. 

In GOBL, I'm choosing to follow the ABRASF way of encoding this information and that's why I'm listing it as the only source. We could list both sources, or maybe just the law, instead of the formats, if you think it's better. Let me know.